internal/kafka/command_link_describe.go

```diff
@@ -29,10 +29,10 @@ func (c *linkCommand) newDescribeCommand() *cobra.Command {
 		RunE:              c.describe,
 	}
 
-	pcmd.AddForceFlag(cmd)
```

Comment:
Huh, wonder why this was added in the first place.

Reply:
Copy paste from the wrong file.